pkg/helmer/helmer.go

```diff
@@ -30,15 +33,21 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
-func DefaultSettings() *cli.EnvSettings {
+func DefaultSettings() (*cli.EnvSettings, error) {
 	s := cli.New()
 
-	s.RepositoryConfig = "/cache/helm/repositories/config.yaml"
-	s.RepositoryCache = "/cache/helm/cache"
+	cacheDir, err := os.UserCacheDir()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to obtain a cache directory")
+	}
+
+	s.RepositoryConfig = filepath.Join(cacheDir, "special-resource-operator/helm/repositories/config.yaml")
+	s.RepositoryCache = filepath.Join(cacheDir, "special-resource-operator/helm/cache")
+	s.RegistryConfig = filepath.Join(cacheDir, "special-resource-operator/helm/registry.json")
```

Comment:
Will those repositories under `cacheDir` be created automatically by Helm?

Reply:
They were previously, so they should be now.
Also, during testing (both in container and outside) there were no issues, and I see that I have `~/.cache/special-resource-operator/helm/{cache,repositories}/` on disk